diff --git a/LLVMExports.cmake b/LLVMExports.cmake
index 1d91d5c..4cc7b87 100644
--- a/LLVMExports.cmake
+++ b/LLVMExports.cmake
@@ -480,7 +480,7 @@ set_target_properties(LLVMDebugInfoCodeView PROPERTIES
 add_library(LLVMDebugInfoPDB STATIC IMPORTED)
 
 set_target_properties(LLVMDebugInfoPDB PROPERTIES
-  INTERFACE_LINK_LIBRARIES "C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/DIA SDK/lib/amd64/diaguids.lib;LLVMBinaryFormat;LLVMObject;LLVMSupport;LLVMDebugInfoCodeView;LLVMDebugInfoMSF"
+  INTERFACE_LINK_LIBRARIES "C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/DIA SDK/lib/amd64/diaguids.lib;LLVMBinaryFormat;LLVMObject;LLVMSupport;LLVMDebugInfoCodeView;LLVMDebugInfoMSF"
 )
 
 # Create imported target LLVMSymbolize
